GPU/Common/DepthBufferCommon.cpp

```diff
@@ -218,14 +218,18 @@ bool FramebufferManagerCommon::ReadbackDepthbuffer(Draw::Framebuffer *fbo, int x
 
 		DepthUB ub{};
 
+		// Setting this to 0.95f eliminates flickering lights with delayed readback in Syphon Filter.
+		// That's pretty ugly though! But we'll need to do that if we're gonna enable delayed readback in those games.
```

Comment:
While it's ugly, tbh if we're sure it's fine and looks about right across the game this is actually a type of game hack / compatibility flag I'm not so against.  Since this is conceptually taking something we know how to do right (blocking) and applying an engine or series specific hack to make it still look the same but run faster.

That doesn't betray the goals of making the emulator capable of running all games well and solving underlying problems.  It is ugly, though.  Not saying it's not ugly.

-[Unknown]

Reply:
It does, but it's only really noticeable if you really look for it. However it's also possible there are some situations with thin walls where lights will shine through now when they didn't before - I didn't find any playing for a while, but it would not surprise me.

So before I do a compat hack for this, I want to see how good we can get it with fully async readbacks (that happen as soon as the GPU performs it, and doesn't wait until a guaranteed safe time the next time around the frame contexts).